pkgs/development/ocaml-modules/ppx_import/default.nix

```diff
@@ -1,28 +1,25 @@
-{ lib, fetchurl, buildDunePackage, ocaml
-, ounit, ppx_deriving, ppx_tools_versioned
-, ppxlib, ocaml-migrate-parsetree
+{ lib, fetchurl, buildDunePackage
+, ppx_tools_versioned
+, ocaml-migrate-parsetree
 }:
 
 buildDunePackage rec {
   pname = "ppx_import";
-  version = "1.7.1";
+  version = "1.8.0";
 
   useDune2 = true;
 
   minimumOCamlVersion = "4.04";
 
   src = fetchurl {
-    url = "https://github.com/ocaml-ppx/ppx_import/releases/download/v${version}/ppx_import-v${version}.tbz";
-    sha256 = "16dyxfb7syz659rqa7yq36ny5vzl7gkqd7f4m6qm2zkjc1gc8j4v";
+    url = "https://github.com/ocaml-ppx/ppx_import/releases/download/v${version}/ppx_import-${version}.tbz";
+    sha256 = "0zqcj70yyp4ik4jc6jz3qs2xhb94vxc6yq9ij0d5cyak28klc3gv";
   };
 
   propagatedBuildInputs = [
-    ppxlib ppx_tools_versioned ocaml-migrate-parsetree
+    ppx_tools_versioned ocaml-migrate-parsetree
   ];
 
-  doCheck = true;
```

Comment:
Doesn't the test suite anymore? Looks unchanged to me.

Reply:
I can’t manage to make it work…

Reply:
I've managed to get it to work, apparently the `ppx_deriving` plugins need propagation of `ppxlib`. The two commits to make it work can be found here: <https://github.com/vbgl/nixpkgs/pull/4>

Reply:
Alright. Thanks for your help. This won’t work with 4.12 yet but this is still a good first step.

Reply:
#116730 makes it work on 4.12 as well, but I'm not sure if we are at a point where we can merge that.